app/src/main/java/org/simple/clinic/editpatient/PatientEditScreen.kt

```diff
@@ -173,6 +178,14 @@ class PatientEditScreen(context: Context, attributeSet: AttributeSet) : Relative
     genderButton.isChecked = true
   }
 
+  fun setPatientAge(age: Int) {
+    ageEditext.setTextAndCursor(age.toString())
+  }
+
+  fun setPatientDateofBirth(dateOfBirth: LocalDate) {
+    dateOfBirthEditText.setTextAndCursor(DATE_OF_BIRTH_FORMAT_FOR_UI.format(dateOfBirth))
```

Comment:
I intentionally don't want to do this because that is presentation logic and doesn't belong in the controller. 

IMO, this will end up getting tested manually because the dates won't get formatted properly.

Reply:
Okay